src/backend/access/gin/ginutil.c

```diff
@@ -29,7 +29,6 @@
 #include "utils/index_selfuncs.h"
 #include "utils/typcache.h"
 
-
```

Comment:
Some times our IDE will delete extra blank like this one. And we usually choose not to change it. 
The Postgres will run pgindent at fixed periods to resolve code style, we will get unnecessary conflicts when Merge to next version. That's painful.